vision_msgs_rviz_plugins/src/bounding_box_3d.cpp

```diff
@@ -0,0 +1,138 @@
+// Copyright 2023 Open Source Robotics Foundation, Inc.
```

Comment:
Are you the open source robotics foundation and/or is this code heavily based on code that is copyrighted to them?

Reply:
No I am not. I thought since I was contributing here it had to be that way?
It's the first time that I contribute to an open source project so I am not sure about the procedure.

Should I put my name?

Reply:
No, this should be you if you're contributing on your own time and resources - or your company otherwise

In general, unless you sign away your rights (commonly known as a Contributor License Agreement, or CLA), when you contribute to an open source project, you still own the copyright but you're giving royalty free and irrevocable control over the code to the repository under the terms of the software license. A CLA might exist so that a repository can re-license the code or do other legal things to it to make it easier. Else, to relicense, you'd need every single copyright holder's approval to relicense (which is unlikely to occur due to sheer logistics). So in many ways, not having a CLA is a nice projection in open-source against someone doing something against the spirit of open source since I, as the maintainer, can't simple wave a wand and start charging people to use this code that you wrote

Reply:
Thank you for pointing that out to me.
Will change it with the next push